keyboards/aleth42/keymaps/default/keymap.c

```diff
@@ -0,0 +1,156 @@
+/* Copyright 2020 monksoffunk
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+    QWERTY = SAFE_RANGE,
+    LOWER,
+    RAISE,
+    ADJUST,
+};
+
+#define L_QWERTY 0
+#define L_LOWER 1 << _LOWER
+#define L_RAISE 1 << _RAISE
+#define L_ADJUST 1 << _ADJUST
```

Comment:
I'm not really a fan of this.

Reply:
Is this the better?

#define L_QWERTY 0
#define L_LOWER 2
#define L_RAISE 4
#define L_ADJUST 8

Reply:
@drashna  How should I do?

Reply:
Sorry, it would be best to just use the layer numbers, and use the `get_highest_layer(state)`.